cves/kernel/CVE-2017-1000252.yml

```diff
@@ -75,7 +75,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1490781]
```

Comment:
The "Bugs" section is for links to bug reports.

Reply:
Thanks I will update!